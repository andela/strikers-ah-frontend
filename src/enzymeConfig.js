import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import 'jest-localstorage-mock';

export default Enzyme.configure({
  adapter: new Adapter(),
});
