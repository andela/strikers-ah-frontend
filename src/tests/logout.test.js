import { shallow } from 'enzyme';
import React from 'react';
import '../enzymeConfig';
import Logout from '../components/logout';

describe('TEST LOGOUT COMPONENT', () => {
  test('it should logout user', () => {
    const wrapper = shallow(<Logout />);
    expect(wrapper).toMatchSnapshot();
  });
});
