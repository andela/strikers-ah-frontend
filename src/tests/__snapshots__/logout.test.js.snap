// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TEST LOGOUT COMPONENT it should logout user 1`] = `ShallowWrapper {}`;
